frontend/src/enums/loader.ts

```diff
@@ -0,0 +1,7 @@
+export enum ELoader {
+  page = 'page',
+  content = 'content',
+  component = 'component',
+}
+
+export type TLoaderVaariants = `${ELoader}`;

```

Comment:
```suggestion
export type TLoaderVariants = `${ELoader}`;

```
and what is the purpose of using `${ELoader}`?